.github/workflows/simulation.yml

```diff
@@ -204,6 +204,7 @@ jobs:
       uses: codecov/codecov-action@v1
 
   external-commissioner:
+    continue-on-error: true
```

Comment:
I believe GitHub Actions will always mark this test with a green check. The only way to see if this test fails is by digging into the logs. Is there something more visible?

Reply:
Just some random thoughts: if we have a comment bot, we can use it to comment in the PR and @ related people.

Reply:
Failed tests will be shown like this: https://github.com/Moandor-y/openthread/actions/runs/166971709. The workflow succeeded but it still shows the job failure. So I guess in the PR UI it will show something like "58 successful, 1 failing, 1 expected, and 2 neutral checks", we don't have to dig into the logs.

Reply:
Tried 2 options:
1. Add `continue-on-error` to `jobs`: This looks the same as before. The check fails, and has a red cross icon in the PR.
1. Add `continue-on-error` to `steps`: This makes all things green. The only way to see failures is by logs.
I think it's better to use option 2 with something making it more visible.

Reply:
@Moandor-y , do you have a proposal for "something making it more visible"?

Reply:
I think commenting on PR is a good idea. I'll find a way to do this.

Reply:
I investigated on commenting on PR. Basically we need to build a simple GitHub App similar to `openthread-size-report` for doing this.

Reply:
I think we can remove dependency on ot-br-posix by using openthread-daemon. https://github.com/openthread/openthread/tree/master/src/posix

Reply:
> I think we can remove dependency on ot-br-posix by using openthread-daemon.

Agree.

Reply:
Created https://github.com/openthread/ot-commissioner/pull/127 to remove the dependency.